src/app/home/home.component.ts

```diff
@@ -15,13 +15,19 @@ import { Fabric8UIConfig } from '../shared/config/fabric8-ui-config';
 import fabric8Logo from '../../assets/images/fabric8_logo.png';
 import openshiftLogo from '../../assets/images/OpenShift-io_logo.png';
 
+import { FeatureTogglesService } from '../feature-flag/service/feature-toggles.service';
+
 @Component({
   encapsulation: ViewEncapsulation.None,
   selector: 'alm-home',
   templateUrl: './home.component.html',
   styleUrls: ['./home.component.less']
 })
 export class HomeComponent implements OnInit, OnDestroy {
+  newHomeDashboardEnabled: boolean = false;
+  myInterval: number = 0;
+  noWrapSlides: boolean = true;
```

Comment:
Nothing wrong with following the example. However, the carousel can be simplified to just this:

&lt;carousel interval="0" noWrap="true"&gt;

Reply:
all cleaned up